napari/layers/graph/graph.py

```diff
@@ -426,6 +445,15 @@ def _remove_nodes(
         is_buffer_domain : bool
             Indicates if node indices are on world or buffer domain.
         """
+        self.events.data(
+            value=self.data,
+            action=ActionType.REMOVING,
+            data_indices=tuple(
+                    self.selected_data,
+                ),
+                vertex_indices=((),),
```

Comment:
Formatting looks a little odd here - did `pre-commit` run on these files? I think `vertex_indices` should be de-indented one block